litmus-portal/frontend/src/components/ToggleComponent/index.tsx

```diff
@@ -0,0 +1,70 @@
+import React from "react";
+import ToggleButton from "@material-ui/lab/ToggleButton";
+import ToggleButtonGroup from "@material-ui/lab/ToggleButtonGroup";
+import { Typography } from "@material-ui/core";
+
+export default function ToggleComponent() {
+	const [alignment, setAlignment] = React.useState<string | null>("pass");
+
+	const handleAlignment = (
+		event: React.MouseEvent<HTMLElement>,
+		newAlignment: string | null
+	) => {
+		setAlignment(newAlignment);
+	};
+	return (
+		<ToggleButtonGroup
+			value={alignment}
+			exclusive
+			onChange={handleAlignment}
+			aria-label="text alignment"
+		>
+			<ToggleButton
```

Comment:
Can we breakout ToggleButton into a component, Use normal styling instead of inline styling and pass the alignment property as props. We can access props in useStyles like so: https://material-ui.com/styles/api/#arguments-3

Reply:
Will send the new component in the next PR, working on it